sdk/keyvault/azsecrets/go.mod

```diff
@@ -2,9 +2,13 @@ module github.com/Azure/azure-sdk-for-go/sdk/keyvault/azsecrets
 
 go 1.16
 
+replace github.com/Azure/azure-sdk-for-go/sdk/internal => ../../internal
+
+replace github.com/Azure/azure-sdk-for-go/sdk/azidentity => ../../azidentity
+
 require (
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.19.0
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v0.20.0
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v0.11.0
```

Comment:
I assume this will wait for the upcoming v0.12.0 release.

Reply:
Yes, after the azidentity release I can bump to v0.12.0 and remove that directive as well.